src/utils/shuffleArray.js

```diff
@@ -0,0 +1,16 @@
+// Fisher-Yates (aka Knuth) Shuffle.
```

Comment:
suggestion: can use sort function to shuffle array
ex - 
const shuffledArr = arr.sort((a, b) => [1, -1][Math.floor(Math.random() * 2)]);

Reply:
Yes, I actually came across this too, but there are a few hiccups with this approach. I am linking a few articles which suggest fisher-yates algorithm is a better one.
1. https://javascript.info/task/shuffle
2. https://dev.to/codebubb/how-to-shuffle-an-array-in-javascript-2ikj

Reply:
Thanks for the articles . Went through it . Yep agree fisher-yates is the better one.